chore: Document workaround for sending cookies services to datalayer (#681)
front-commerce/developers.front-commerce.com: versioned_docs/version-2.x/advanced/analytics/plugins.mdx

</Figure>

> #### Retrieve authorized cookie services in GTM's datalayer
> 
> If your project needs to retrieve the authorized cookies services in GTM's
> datalayer, in example for triggering other analytics services, you can do so by
> [declaring them in your `analytics.js`](./getting-started#retrieve-authorized-cookie-services-in-analytics).

Reviewer: **nitpick:** it feels like a complement to the previous explanation

**suggestion:** 
```suggestion
Please note that to retrieve the authorized cookies services in GTM's
datalayer, services must be [declared in your `analytics.js`](./getting-started#retrieve-authorized-cookie-services-in-analytics).
```

contribute to them if you have any ideas! 💡

:::

> ## Known issue

Reviewer: **issue:** several issues here IMO
1. this isn't an issue per se
2. datalayer is specific to GTM, so it shouldn't be mentioned for this part
3. it is more related to the signature of `settings()`

**suggestion:** rework this MR to highlight this in https://deploy-preview-681--heuristic-almeida-1a1f35.netlify.app/docs/2.x/advanced/analytics/getting-started#configure-analytics and maybe add a note to the new section from GTM's example

Author: I added a section in getting started to mention the workaround, removed the issue mention, and updated GTM's documentation to mention it.

Please let me know if the wording isn't right still.
